application/helpers/SurveyRuntimeHelper.php

```diff
@@ -1208,7 +1207,13 @@ private function moveSubmitIfNeeded()
             if(!$surveyActive) {
                 $this->aSurveyInfo['include_content'] = 'submit_preview';
             }
-            Yii::app()->twigRenderer->renderTemplateFromFile("layout_global.twig", array('oSurvey'=> $oSurvey, 'aSurveyInfo'=>$this->aSurveyInfo), false);
+            $sHtml = Yii::app()->twigRenderer->renderTemplateFromFile("layout_global.twig", array('oSurvey'=> $oSurvey, 'aSurveyInfo'=>$this->aSurveyInfo), true);
+            $oTemplate = Template::model()->getInstance();
```

Comment:
I test with default template only, think it take the current one (related to survey)